packages/terra-progress-bar/src/ProgressBar.jsx

```diff
@@ -23,21 +23,28 @@ const propTypes = {
    * Sets the maximum possible fill-value.
    */
   max: PropTypes.number,
+  /**
+   * Boolean check for gradient.
+   */
+  gradient: PropTypes.bool,
 };
 
 const defaultProps = {
   heightSize: 'small',
   max: 100,
   value: 0,
+  gradient: false,
 };
 
 const ProgressBar = ({
   heightSize,
   value,
   max,
+  gradient,
   ...customProps
 }) => {
   const classes = cx([
+    gradient && 'progress-bar-gradient',
```

Comment:
ClassNames provides first class support for logic statements. 

```
// Add this class if gradient is true
{ 'progress-bar-gradient': gradient },
```

Reply:
Thanks. Updated :)